.github/workflows/publish-prisma-fmt-wasm.yml

```diff
@@ -0,0 +1,68 @@
+name: Build and publish @prisma/prisma-fmt-wasm
+
+concurrency: build-prisma-fmt-wasm
+
+on:
+  workflow_dispatch:
+    inputs:
+      enginesWrapperVersion:
+        required: true
+      enginesHash:
+        required: true
+      npmDistTag:
+        required: true
+        default: 'latest'
+
+jobs:
+  build:
+    name: Build and publish @prisma/prisma-fmt-wasm
+    runs-on: ubuntu-latest
+    steps:
+      - name: Print input
+        env:
+          THE_INPUT: '${{ toJson(github.event.inputs) }}'
+        run: |
+          echo $THE_INPUT
+
+      - uses: actions/checkout@v3
+        ref: ${{ github.event.inputs.enginesHash }}
+      - uses: cachix/install-nix-action@v16
+
+      #
+      # Build
+      #
+
+      - run: nix build .#prisma-fmt-wasm
+
+      #
+      # Publish
+      #
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+
+      - name: Set up NPM token
+        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+
+      - run: |
+          PACKAGE_DIR=$( nix run .#renderPrismaFmtWasmPackage ${{ github.event.inputs.enginesWrapperVersion }})
+          npm publish "$PACKAGE_DIR" --access public --tag ${{ github.event.inputs.npmDistTag }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      #
+      # Failure handlers
+      #
+
+      - name: Set current job url in SLACK_FOOTER env var
+        if: ${{ failure() }}
+        run: echo "SLACK_FOOTER=<$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID|Click here to go to the job logs>" >> $GITHUB_ENV
+
+      - name: Slack Notification on Failure
+        if: ${{ failure() }}
+        uses: rtCamp/action-slack-notify@v2.2.0
+        env:
+          SLACK_TITLE: 'prisma-fmt-wasm publishing failed :x:'
+          SLACK_COLOR: '#FF0000'
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
Note: Let's check that `SLACK_WEBHOOK` is set in the settings (needs admin rights though)

Reply:
Not in https://github.com/prisma/prisma-engines/settings/secrets/actions

Reply:
I don't have admin rights on this repo. Could you copy over the values from prisma/prisma-fmt-wasm? (or point me to someone with the permission)

Reply:
Those are secrets, no copying over. We need to find the original values that were put in there and then I am happy to make sure they are present here as well. (Although having an Npm token in here is def new and kinda risky @Jolg42 )

Reply:
I'll try to see how to make slack webhooks. This one is for #feed-prisma-fmt-wasm-publish-failures

Reply:
We have an app on Slack for that, I'll send the link to Jan so he can do that